onnxruntime/core/providers/xnnpack/detail/utils.cc

```diff
@@ -2,16 +2,19 @@
 // Licensed under the MIT License.
 
 #include "utils.h"
+#include <set>
```

Comment:
<unordered_set> should be included instead of <set> by the instance of `default_supported_types`